db-drda/src/main/java/com/splicemachine/db/impl/drda/DRDAConnThread.java

```diff
@@ -380,6 +393,8 @@ public void run() {
                 // TODO: Could make use of Throwable.addSuppressed here when
                 //       compiled as Java 7 (or newer).
                 try {
+                    debug("encountered error " + error.getMessage());
```

Comment:
Use `LOG.debug(message, error)`. It will also print the `error.getMessage()` and the stack trace.

Reply:
done.